_test/build_logger_test.js

```diff
@@ -0,0 +1,123 @@
+/* jshint node: true */
+/* jshint expr: true */
+/* jshint mocha: true */
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+var chai = require('chai');
+var sinon = require('sinon');
+var buildLogger = require('../build-logger.js');
+
+var expect = chai.expect;
+chai.should();
+
+describe('BuildLogger', function() {
+  var logger, logFileDir, logFilePath;
+
+  before(function() {
+    logFileDir = path.resolve(__dirname, 'buildLogger_test');
+    logFilePath = path.resolve(logFileDir, 'build.log');
+  });
+
+  beforeEach(function(done) {
+    fs.exists(logFileDir, function(exists) {
+      (exists ? fs.chmod : fs.mkdir)(logFileDir, '0700', done);
+    });
+  });
+
+  afterEach(function(done) {
+    fs.exists(logFilePath, function(exists) {
+      if (exists) { fs.unlink(logFilePath, done); } else { done(); }
+    });
+  });
+
+  after(function(done) {
+    fs.exists(logFileDir, function(exists) {
+      if (exists) { fs.rmdir(logFileDir, done); } else { done(); }
+    });
+  });
+
+  var makeLogger = function(done) {
+    return new buildLogger.BuildLogger(logFilePath, done);
+  };
+
+  var captureLogs = function() {
+    sinon.stub(console, 'log').returns(void 0);
```

Comment:
I don't think you need to use `void` here. It's not really a pattern in JS. I would just use `null`.